Resources/doc/internals/configuring_the_http_client.md

```diff
@@ -27,4 +27,93 @@ hwi_oauth:
 
 If you don't want to use the HTTPlug Bundle, the client and factory services **must be specified**.
 
+## Difference since 0.6+ version
+
+On v0.5 and below, you was able to configure your http client like that:
+
+```yaml
+# app/config/config.yml
+
+hwi_oauth:
+    http_client:
+        timeout:       10 # Time in seconds, after library will shutdown request, by default: 5
+        verify_peer:   false # Setting allowing you to turn off SSL verification, by default: true
+        ignore_errors: false # Setting allowing you to easier debug request errors, by default: true
+        max_redirects: 1 # Number of HTTP redirection request after which library will shutdown request,
+                         # by default: 5
+        proxy: "example.com:8080" # String with proxy configuration for cURL connections, ignored by default.
+                                    # "" -> don't set proxy and will use proxy system
+                                    # "example.com:8080" -> set custom proxy
+                                    # ":" -> disable proxy usage, ignoring also proxy system and ENVIRONMENT variables
+```
+
+As we not use Httplug, the http configuration does not rely from our bundle anymore.
+Still, you can always configure Guzzle6 (if you use this client) thanks to the
+[Guzzle6 adapter](http://docs.php-http.org/en/latest/clients/guzzle6-adapter.html).
+
+### Configure with HttplugBundle
+
+You need to [install and configure](../1-setting_up_the_bundle.md) the HttplugBundle first.
+
+Then, the configuration is quite the same:
```

Comment:
Then apply the following configuration (which in fact is quite similar to what we had in the previous section) ?

Reply:
> which in fact is quite similar to what we had in the previous section

Yeah, this is just an example to show clearly the differences.